lib/kino.ex

```diff
@@ -116,20 +116,35 @@ defmodule Kino do
   `inspect/2`.
   '''
 
+  import Kernel, except: [inspect: 1]
+
   @type nothing :: :"do not show this result in output"
 
   @doc """
-  Sends the given term as cell output.
+  Renders the given term as cell output.
 
-  This allows any Livebook cell to have multiple evaluation
-  results. You can think of this function as a generalized
-  `IO.puts/2` that works for any type.
+  This effectively allows any Livebook cell to have multiple
+  evaluation results.
   """
-  @spec render(term()) :: nothing()
+  @spec render(term()) :: term()
   def render(term) do
     output = Kino.Render.to_livebook(term)
     Kino.Bridge.put_output(output)
-    nothing()
+    term
+  end
+
+  @doc """
+  Inspects the given term as cell output.
+
+  This works essentially the same as `IO.inspect/2`, except it
+  always produces colored text and respects the configuration
+  set with `configure/1`.
+  """
+  @spec inscpect(term()) :: term()
+  def inscpect(term) do
+    output = Kino.Output.inspect(term)
+    Kino.Bridge.put_output(output)
+    term
```

Comment:
Can we add some basic tests? We can probably use `capture_io` to check that it is inspected and then itself is returned.

Reply:
`capture_io` only captures to string, but we would need to mock the group leader to handle various messages and that actually tests `Kino.Bridge` instead. We may need some abstraction for testing the bridge since it started to grow, but maybe handle this separately?

Reply:
I think only capturing the string and not the registration would be fine as a starting point. Especially because we need to start testing which parts of Kino works with regular IO group leader, beyond the mock group leader too. So I would add the io group leader test now, do the mocking later.

Reply:
My point is that capturing just the output doesn't work in this case, since we send a `{:livebook_put_output, _}` request to the group leader.

Reply:
@josevalim I've just pushed a simple test for now, let me know what you think :)

Reply:
@jonatanklosko that's what I mean though, Kino.inspect should totally work even on a regular group leader. :)

Reply:
But yeah, we can work on making this work later. :)

Reply:
That's something separate though, we are yet to add the terminal compatibility layer. It would basically convert all outputs to text or ignored, while currently it is `{:text, _}`.